stock_inventory_revaluation/wizards/stock_inventory_revaluation_mass_post.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, models, _, exceptions
+
+
+class StockInventoryRevaluationMassPost(models.TransientModel):
+
+    _name = 'stock.inventory.revaluation.mass.post'
+    _description = 'Post multiple inventory revaluations'
+
+    @api.model
+    def default_get(self, fields):
```

Comment:
This allows you to select multiple stock.inventory.revaluation lines and post them all at once. It is used, and should not be removed.